src/components/landing/StatusCard.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Card from '@material-tailwind/react/Card';
+import CardBody from '@material-tailwind/react/CardBody';
+import Icon from '@material-tailwind/react/Icon';
+import H6 from '@material-tailwind/react/Heading6';
+import Paragraph from '@material-tailwind/react/Paragraph';
+
+export default function StatusCard({ color, icon, title, children }) {
+    return (
+        <div className="w-full md:w-4/12 px-4 flex justify-center text-center">
+            <Card>
+                <CardBody>
+                    <div
+                        className={`text-white p-3 text-center inline-flex items-center justify-center w-12 h-12 mb-5 shadow-md rounded-full bg-${color}-500`}
```

Comment:
I don't really recommend using background classes like so, actually Tailwind CSS does not recommend using dynamic classes like so.
What we should do here, si we should have a constant with all colors that we want for this component:
```
const colors = {
    "red": "bg-red-500",
    "lightBlue": "bg-lightBlue-500",
    ... etc. you get the point
}
```
and after that, instead of `bg-${color}-500` we would say `${colors[color]}`